ansible/roles/centos7/dashboards/vars/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# vars file for elasticsearch
```

Comment:
Can we name all instances of `elasticsearch` to `opensearch` and `kibana` to dashboards here?

Reply:
ok

Reply:
@TheAlgo  Done the changes.